File: 1833-find-the-highest-altitude/1833-find-the-highest-altitude.ts
function largestAltitude(gain: number[]): number {
    let currentAlt = 0, highestAlt = 0
    for(const alt of gain){
        currentAlt += alt
        if(currentAlt > highestAlt) highestAlt = currentAlt
    }
    return highestAlt
};

File: 1833-find-the-highest-altitude/1833-find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        currentAlt, highestAlt = 0,0
        for alt in gain:
            currentAlt += alt
            if highestAlt < currentAlt:
                highestAlt = currentAlt
        return highestAlt